src/EditorFeatures/Test/RenameTracking/RenameTrackingTestState.cs

```diff
@@ -112,14 +112,12 @@ internal sealed class RenameTrackingTestState : IDisposable
             if (languageName == LanguageNames.CSharp)
             {
                 _codeFixProvider = new CSharpRenameTrackingCodeFixProvider(
-                    Workspace.ExportProvider.GetExport<Host.IWaitIndicator>().Value,
```

Comment:
I didn't understand this change (not familiar enough with area or logic) so I can't comment.